app/src/main/java/com/jerboa/ui/components/community/CommunityViewModel.kt

```diff
@@ -144,6 +145,8 @@ class CommunityViewModel : ViewModel(), Initializable {
 
                 else -> {}
             }
+
+            onComplete()
```

Comment:
I would only do it on Success, there is no point on updating if the following has failed